src/LayerPlan.cpp

```diff
@@ -459,7 +459,15 @@ GCodePath& LayerPlan::addTravel(Point p, bool force_comb_retract)
             }
         }
     }
-    
+
+    // CURA-6675:
+    // Retraction Minimal Travel Distance should work for all travel moves. If the travel move is larger than the
```

Comment:
comment is wrong: move is shorter not larger

Reply:
Fixed in de1f4506